js/assessments/textLengthAssessment.js

```diff
@@ -70,9 +91,15 @@ var calculateWordCountResult = function( wordCount, i18n ) {
 			text: i18n.dngettext(
 				"js-text-analysis",
 				/* Translators: %1$d expands to the number of words in the text */
-				"The text contains %1$d word, which is far too low. Increase the word count.",
-				"The text contains %1$d words, which is far too low. Increase the word count.",
+				"The text contains %1$d word.",
+				"The text contains %1$d words.",
 				wordCount
+			) + " " + i18n.dngettext(
+				"js-text-analysis",
+				/* Translators: The preceding sentence is "The text contains x words.", %2$s expands to the recommended minimum of words */
+				"This is far below the recommended minimum of %2$d word. Increase the word count with content that is relevant for the topic.",
```

Comment:
Can we make this string equal to https://github.com/Yoast/YoastSEO.js/pull/706/files#diff-06a348390e9b414dffad37921862c1d1R81? 
They both contain the same message, and making them identical saves work for our translators.